### ISSUE TITLE
No error for PDF component when exceeding max_file_size param of gr.Blocks 

### ISSUE THREAD

**Issue by Jocelyne-Smith** [freddyaboulton/gradio-pdf#19]
When the max_file_size param for gr.Blocks is set, all components except the PDF component responds with:

![image](https://github.com/user-attachments/assets/5d15bc69-b9d4-4a7b-acba-c3c14c4e2ed7)

The PDF component simply refuses to upload the file and the user is left wondering if the component is broken without a proper error message.

```python
from gradio_pdf import PDF
import gradio as gr

with gr.Blocks() as interface:

    pdf_filepath = PDF(label="Specifications (.pdf)",
                              interactive=True)

if __name__ == "__main__":
    interface.launch(debug=True, max_file_size=200)

```

**Comment by freddyaboulton** [comment]
Should have a fix out this week @Jocelyne-Smith

**Comment by freddyaboulton** [comment]
Fixed in version 0.0.14!
